test/widgets/action_sheet_test.dart

```diff
@@ -38,11 +38,12 @@ late FakeApiConnection connection;
 Future<void> setupToMessageActionSheet(WidgetTester tester, {
   required Message message,
   required Narrow narrow,
+  Account? account,
 }) async {
```

Comment:
Add parameter Account so that we can override realmUrl to produce Url without '/' before '#'.